pkg/scheduler/ugm/manager.go

```diff
@@ -73,11 +80,66 @@ func (m *Manager) IncreaseTrackedResource(queuePath string, applicationID string
 		return fmt.Errorf("mandatory parameters are missing. queuepath: %s, application id: %s, resource usage: %s, user: %s",
 			queuePath, applicationID, usage.String(), user.User)
 	}
-	m.lock.Lock()
-	defer m.lock.Unlock()
+	m.Lock()
+	defer m.Unlock()
 	var userTracker *UserTracker
 	if m.userTrackers[user.User] == nil {
-		userTracker = newUserTracker(user)
+		userTracker = newUserTracker(user.User)
+
+		// Set the limits for all configured queue paths of the user
+		for configQueuePath, config := range m.userLimitsConfig[user.User] {
+			log.Logger().Debug("Setting the limit max applications settings.",
+				zap.String("user", user.User),
+				zap.String("queue path", configQueuePath))
+			maxApps, maxAppsErr := config[maxapplications].(uint64)
+			if !maxAppsErr {
+				log.Logger().Warn("Problem in setting the limit max applications settings. Unable to cast the value from interface to uint64",
+					zap.String("user", user.User),
+					zap.String("queue path", configQueuePath),
+					zap.Uint64("limit max applications", maxApps))
+				return fmt.Errorf("unable to set the max resources. user: %s, queuepath : %s, applicationid: %s",
+					user.User, configQueuePath, applicationID)
+			}
+			err := userTracker.setMaxApplications(maxApps, configQueuePath)
+			if err != nil {
+				log.Logger().Warn("Problem in setting the limit max applications settings.",
+					zap.String("user", user.User),
+					zap.String("queue path", configQueuePath),
+					zap.Uint64("limit max applications", maxApps),
+					zap.Error(err))
+				return fmt.Errorf("unable to set the max applications. user: %s, queuepath : %s, applicationid: %s, usage: %s, reason: %w",
+					user.User, configQueuePath, applicationID, usage.String(), err)
+			}
+			maxResources, maxResourcesErr := config[maxresources].(map[string]string)
```

Comment:
Nit: just simply `ok` instead of  `maxResourcesErr`, that's the idiomatic Go approach.